src/Tribe/Validator/Base.php

```diff
@@ -7,6 +7,24 @@
  */
 class Tribe__Validator__Base implements Tribe__Validator__Interface {
 
+	/**
+	 * @param mixed $value
+	 *
+	 * @return bool
+	 */
+	public function is_not_null( $value ) {
+		return null !== $value;
```

Comment:
How about: 

```php
return ! $this->is_null( $value );
```

Reply:
That would be cool but I feel it unnecessarily adds to the call stack.